### ISSUE TITLE
Error message in loading models with OMEGA

### ISSUE THREAD

**Issue by DuyTran16** [mrgsolve/learn#2]
I was wondering if you can help me figure out why an error message appears when reading models with OMEGA 
For example
mod_mero <- mread(model = "meropenem", project = "model")

Error in c(<S4 object of class "omegalist">) : could not find symbol "recursive" in environment of the generic function

**Comment by kylebaron** [comment]
Hi @DuyTran16  - 

The installed package is corrupted.  Try re-installing the package.  You *might* have to go in and manually delete the installed package first.  I'd just try re-installing first.

**Comment by DuyTran16** [comment]
@kylebmetrum 
The package is v 0.8.10 on my Mac computer.
I tried to remove the package by
`remove.packages("mrgsolve")`
and then re-install by
`install.packages("mrgsolve")`

It still show the same error message.  I then locate the directory of the package 
`path.package("mrgsolve")`
and manually deleted it, reinstall the package again, and ran the model but still resulted with same errors.

I tried using mrgsolve on Windows computer and it worked successfully.  I am not sure why it doesn't work on the Mac. I have used mrgsolve to reproduce models from Berkeley Madonna code with success on Mac but the code doesn't have $OMEGA block.

**Comment by kylebaron** [comment]
Not sure what's going on there, but the error means the package install is corrupt.  Are you installing the source or binary?

Also ... are you going to be at ASCPT?  If so we could meet up there and look.

**Comment by DuyTran16** [comment]
I tried to reinstall again but using developmental version (v 0.8.10.9009) instead of CRAN
`devtools::install_github("metrumresearchgroup/mrgsolve")`
and it worked on the Mac.

Only error with meropenem code was on line 50, which was missing an end parenthesis after TVV2, but after this was fixed the model ran successfully. 
`double V2 =  exp(log(TVV2) +  ETA(4));`

Thanks for you help!  I am not attending ASCPT, but hopefully will attend ACOP9 in the fall.
